packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -129,34 +178,48 @@ export async function makeCustomSentryVitePlugins(options?: CustomSentryVitePlug
     __sentry_sveltekit_output_dir: outputDir,
   };
 
-  const customDebugIdUploadPlugin: Plugin = {
-    name: 'sentry-sveltekit-debug-id-upload-plugin',
+  const sourceMapSettingsPlugin: Plugin = {
```

Comment:
l: I'm totally fine with making this its own plugin but is there a specific reason why it needs to be its own plugin? To be clear, even if there's none, let's leave it as it's nice separation of concerns. Just curious :)

Reply:
The reason I did this was really just separation of concerns. It previously was inside the custom debug id plugin and maybe that's alright too but I assumed those are two different things.